Header header

#Массив значений внешнего АЦП [Вольт] [0 - 2.5 В]
float64[6] values